packages/lesswrong/components/common/LWPopper.tsx

```diff
@@ -84,55 +98,74 @@ const LWPopper = ({
     }
   ];
 
+  // Custom modifier to check for a second tooltip
+  const secondTooltipModifier = checkForSecondTooltip ? [
+    {
+      name: 'checkSecondTooltip',
+      enabled: true,
+      phase: 'main',
+      fn({ state }: { state: State }) {
+        // Logic to check for a second tooltip below the anchor element
+        const secondTooltipExists = document.querySelector(SECOND_TOOLTIP_SELECTOR); // Replace with actual selector
+        if (secondTooltipExists) {
+          // Adjust placement to 'left-end' if a second tooltip is detected
+          state.placement = 'left-end';
+        }
+      },
+    }
+  ] : [];
+
   const { styles, attributes, update } = usePopper(anchorEl, popperElement, {
     placement,
     modifiers: [
       {
         name: 'computeStyles',
         options: {
-          gpuAcceleration: false, // true by default
+          gpuAcceleration: false,
         },
       },
       ...flipModifier,
-      ...preventOverflowModifier
+      ...preventOverflowModifier,
+      ...secondTooltipModifier,
     ],
   });
 
   if (updateRef && update) {
-    updateRef.current = update
+    updateRef.current = update;
   }
 
-  if (!open)
-    return null;
-  
+  if (!open) return null;
+
   // In some cases, interacting with something inside a popper will cause a rerender that detaches the anchorEl
   // This happened in hovers on in-line reacts, and the button to create a new react ended up on the top-left corner of the page
   if (anchorEl && !anchorEl.isConnected) {
     return null;
   }
-  
-  // We use createPortal here to avoid having to deal with overflow problems and styling from the current child
-  // context, by placing the Popper element directly into the document root
-  // Rest of usage from https://popper.js.org/react-popper/v2/
-  return <>{
-    createPortal(
-      <div
-        ref={setPopperElement}
-        className={classNames({
-          [classes.tooltip]: tooltip,
-          [classes.default]: !tooltip,
-          [classes.noMouseEvents]: !clickable,
-          [classes.hideOnTouchScreens]: hideOnTouchScreens},
-          className
-        )}
-        style={styles.popper}
-        {...attributes.popper}
-      >
-        { children }
-      </div>,
-      document.body
-    )
-  }</>
+
```

Comment:
```suggestion

  // We use createPortal here to avoid having to deal with overflow problems and styling from the current child
  // context, by placing the Popper element directly into the document root
  // Rest of usage from https://popper.js.org/react-popper/v2/
```